{{cookiecutter.project_slug}}/home/migrations/0002_load_initial_data.py

```diff
@@ -22,11 +22,13 @@ def create_homepage(apps, schema_editor):
 
 def create_site(apps, schema_editor):
     Site = apps.get_model("sites", "Site")
+    custom_domain = "{{cookiecutter.custom_domain}}"
 
     site_params = {
         "name": "{{cookiecutter.project_name}}",
-        "domain": "{{cookiecutter.custom_domain}}",
     }
+    if custom_domain:
+        site_params.update({"domain": custom_domain})
```

Comment:
This can be simplified to `site_params["domain"] = custom_domain`

Reply:
True but it comes down to personal preference, I guess. In this instance we are adding only a key value pair so I'll make the change.